File: cloud/dao/system_post.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SystemPost 职位 system_post
type SystemPost struct {
	Id             *int64        `db:"id" json:"id"`                           //bigint 职位ID,PRI
	Name           *string       `db:"name" json:"name"`                       //varchar 职位名称
	Sort           *int32        `db:"sort" json:"sort"`                       //int 显示顺序
	Status         *int32        `db:"status" json:"status"`                   //tinyint 状态
	Deleted        *int32        `db:"deleted" json:"deleted"`                 //tinyint 是否删除
	SystemTenantId *int64        `db:"system_tenant_id" json:"systemTenantId"` //bigint 租户ID
	Creator        null.String   `db:"creator" json:"creator"`                 //varchar 创建人
	CreateTime     null.DateTime `db:"create_time" json:"createTime"`          //datetime 创建时间
	Updater        null.String   `db:"updater" json:"updater"`                 //varchar 更新人
	UpdateTime     null.DateTime `db:"update_time" json:"updateTime"`          //datetime 更新时间
}

File: admin/src/api/interface/systemPost.ts
// system_post 职位
import { ReqPage } from "./index";
export namespace SystemPost {
  export interface ReqSystemPostList extends ReqPage {
    deleted?: number; // 是否删除
    status?: number; // 状态
    name?: string; // 职位名称
  }
  export interface ResSystemPostItem {
    id: number; // 职位ID
    name: string; // 职位名称
    sort: number; // 显示顺序
    status: number; // 状态
    deleted: number; // 是否删除
    tenantId: number; // 租户ID
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}
